mobile/src/androidTest/java/ch/epfl/sdp/mobile/test/state/Dsl.kt

```diff
@@ -101,14 +103,14 @@ suspend fun ComposeContentTestRule.setContentWithTestEnvironment(
     recognizer: SpeechRecognizerFactory = FailingSpeechRecognizerFactory,
     dataStoreFactory: DataStoreFactory = emptyDataStoreFactory(),
     timeProvider: TimeProvider = FakeTimeProvider,
-    strings: LocalizedStrings = English,
     content: @Composable TestEnvironment.() -> Unit,
 ): TestEnvironment {
   val authenticationFacade = AuthenticationFacade(auth, store)
   val socialFacade = SocialFacade(auth, store)
   val chessFacade = ChessFacade(auth, store, assets)
   val speechFacade = SpeechFacade(recognizer)
   val tournamentFacade = TournamentFacade(auth, dataStoreFactory, store, timeProvider)
+  val settingsFacade = SettingsFacade(dataStoreFactory)
   authenticationFacade.signInWithEmail(DefaultEmail, DefaultPassword)
   val user = authenticationFacade.awaitAuthenticatedUser()
   val environment =
```

Comment:
You must add the `SettingsFacade` to the `Facades` object of the `TestEnvironment`.

Reply:
thx